common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -338,7 +338,7 @@ def manage_video_subtitles_save(item, user, old_metadata=None, generate_translat
 
     # 1.
     html5_ids = get_html5_ids(item.html5_sources)
-    possible_video_id_list = [item.youtube_id_1_0] + html5_ids
+    possible_video_id_list = html5_ids + [item.youtube_id_1_0]
```

Comment:
Are subtitles prioritised in the the manner they appear in `possible_video_id_list` ?